### PR TITLE
Fix order book infinite loading

### PR DESCRIPTION
**Pull request by takenagain**
- Fix infinite loading issue in orderbook 
- Remove segwit pairs from dex receive list

### CODE REVIEW

**Review by CharlVS — COMMENTED** [review]
Thanks for the contribution.

Approved, pending QA's sign-off that the changes fix the bug.

**Review by smk762 — APPROVED** [review]
Working here thanks! As noted, some delay for larger orderbooks, but it does appear soon enough (we might look at optimising this later). For smaller orderbooks its there very fast

**Review by CharlVS — APPROVED** [review]

**Pull request merged**
